Mathlib/Analysis/Normed/Ring/Basic.lean

```diff
@@ -680,16 +680,40 @@ variable [SeminormedAddCommGroup α] [Mul α] [NormMulClass α] (a b : α)
 
 end SeminormedAddCommGroup
 
+section NormedAddCommGroup
+
+variable [NormedAddCommGroup α] [MulOneClass α] [NormMulClass α] [Nontrivial α]
+
+/-- Deduce `NormOneClass` from `NormMulClass` under a suitable nontriviality hypothesis. -/
+instance NormMulClass.toNormOneClass : NormOneClass α := by
```

Comment:
This looks risky to me, because we also have [`NormOneClass.nontrivial [NormOneClass A] : Nontrivial A`](https://leanprover-community.github.io/mathlib4_docs/find/?pattern=NormOneClass.nontrivial#doc), which forms a loop.